zx.mjs

```diff
@@ -117,7 +118,8 @@ async function importPath(filepath, origin = filepath) {
   if (ext === '.ts') {
     let {dir, name} = parse(filepath)
     let outFile = join(dir, name + '.mjs')
-    await run`npm_config_yes=true npx tsc --target esnext --module esnext --moduleResolution node ${filepath}`
+    let tsc = await which('tsc').then(() => 'tsc' ).catch(() => 'npm_config_yes=true npx tsc')
```

Comment:
I think nox will do it automatically?

Reply:
`npx <...>` is not equal `npm i -g && node <...>`. You may check this with `--verbose` flag.

Reply:
Binary name and pkg name may not match in general case. So it `npx` performs request every time.

Reply:
But for tsc it’s not a concern, right?